packages/migration/src/migrations/v0017_inlineTextProperties.js

```diff
@@ -128,8 +128,6 @@ function addInlineLetterSpacing(content, letterSpacing) {
   return wrapWithSpan(content, `letter-spacing: ${letterSpacing / 100}em`);
 }
 
-/* eslint-disable security/detect-non-literal-regexp */
```

Comment:
Why was this removed? This file shouldn't be changed.

Reply:
I don't know. Odd. Fixed now - https://github.com/google/web-stories-wp/pull/6244/commits/5d06de165ea669eb143cb3ab7fc115f11e47e922